.github/tests.yaml

```diff
@@ -0,0 +1,31 @@
+betelgeuse: "tests/foreman/ -v -m 'build_sanity'"
+broker[docker]: "tests/foreman/ -v -m 'build_sanity'"
+cryptography: "tests/foreman/ -v -m 'build_sanity'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -v -k 'test_positive_get_links'"
+dynaconf[vault]: "tests/foreman/ -v -m 'build_sanity'"
```

Comment:
Run any tests thats reading secrets from vault instead of build_Sanity.

Reply:
But I dont see any tests using vault package abilities, how could we merge the PR bsaed on tests those are not using that package ?

Reply:
I agree that this package is not used in sanity directly. how are we currently merging this based on which tests ?

Reply:
Make use of any test that's using secrets, compute resource tests needing credentials to interact with public clouds are good candidates.